apps/studio/src/migration/index.js

```diff
@@ -102,7 +102,11 @@ export default class {
       }
       const hasRun = await Manager.checkExists(runner, migration.name)
       if (!hasRun) {
-        await migration.run(runner, this.env)
+        try {
+          await migration.run(runner, this.env)
+        } catch (err) {
+          console.log(`Migration ${migration.name} failed with`, err.name, err.message)
+        }
         await Manager.markExists(runner, migration.name)
```

Comment:
@not-night-but move this inside the `try`?

Reply:
Doh